app/src/main/res/layout/mark_chapters_completed_fragment.xml

```diff
@@ -44,7 +44,7 @@
       <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@color/oppia_primary">
+        android:background="@color/component_color_mark_chapters_completed_secondary_toolbar_color">
```

Comment:
This should be renamed because this is not a toolbar color, maybe `...._container_background_color`

Reply:
> This should be renamed because this is not a toolbar color, maybe `...._container_background_color`

Done